SU-SWS/decanter
# 238 cleanup

# READY FOR REVIEW

# Summary
- Cleaned up file structure and naming conventions
- Updated class name prefixes
- Removed deprecated or no longer being utilized code from project

# Needed By (Date)
- This week

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Compile styleguide (`grunt styleguide`)
3. Ensure that new organization and evident examples align with decisions group made

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- #238 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
